pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -107,25 +117,50 @@ func (r *Retriever) extractSecretItemName(xDescriptor string) string {
 	return strings.ReplaceAll(xDescriptor, fmt.Sprintf("%s:", secretPrefix), "")
 }
 
+// extractConfigMapItemName based in x-descriptor entry, removing prefix in order to keep only the
+// configMap item name.
+func (r *Retriever) extractConfigMapItemName(xDescriptor string) string {
+	return strings.ReplaceAll(xDescriptor, fmt.Sprintf("%s:", configMapPrefix), "")
+}
+
 // readSecret based in secret name and list of items, read a secret from the same namespace informed
 // in plan instance.
 func (r *Retriever) readSecret(name string, items []string) error {
 	logger := r.logger.WithValues("Secret.Name", name, "Secret.Items", items)
 	logger.Info("Reading secret items...")
-
 	secretObj := corev1.Secret{}
 	err := r.client.Get(r.ctx, types.NamespacedName{Namespace: r.plan.Ns, Name: name}, &secretObj)
 	if err != nil {
 		return err
 	}
-
 	logger.Info("Inspecting secret data...")
 	for key, value := range secretObj.Data {
 		logger.WithValues("Secret.Key.Name", key, "Secret.Key.Length", len(value)).
 			Info("Inspecting secret key...")
 		// making sure key name has a secret reference
 		r.store(fmt.Sprintf("secret_%s", key), value)
 	}
+	return nil
+}
+
+//readConfigMap based in configMap name and list of items, read a configMap from the same namespace informed
```

Comment:
Small detail here, we are missing a space on the doc-comment: `// readConfigMap`